### DISCUSSION TITLE
Some wiring questions

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#45] [Q&A] [answered]
Just wiring my DSub and noticed that I have an extra GND to what's on the schematic, presumably due to the odd board I'm using

![IMG_20210624_154415.jpg](https://user-images.githubusercontent.com/17175988/123283478-56094b80-d503-11eb-986d-11af7efa09c9.jpg)

Should I or should I not loop this into the existing ground daisy chain?

Also, while talking wiring, found a description of wire colours for GC connectors (Yellow, 5v, Red, Data, Green Ground, White Ground and Blue 3.3)
Is it fair to assume this will be the same across all controllers/extensions? Colours and number match to mine so assuming so but what's the best way to test?

**Comment by darthcloud** [comment, marked as answer]
NEVER trust cable colors, always check with multimeter.

**Reply by vaderag** [reply]
Fair enough. I figured you'd say that 🤣
Now to find my multi meter in my dog pile of a garage

How about the gnd on the DSub?

**Reply by vaderag** [reply]
Just to update, turned out the colours didn't match... good job I checked!
